ts/screens/main/MessagesScreen.tsx

```diff
@@ -67,26 +88,70 @@ class MessagesScreen extends React.Component<Props, never> {
     ) : null;
   };
 
-  private renderMockedMessages = (
+  public getOrganizationName = (senderServiceId: string): string => {
+    const organizationName = this.props.services.byId[senderServiceId]
+      .organization_name;
+    const departmentName = this.props.services.byId[senderServiceId]
+      .department_name;
+    return `${organizationName} - ${departmentName}`;
+  };
+
+  public renderItem = (messageDetails: IMessageDetails) => {
+    return (
+      <MessageComponent
+        key={messageDetails.item.id}
+        date={messageDetails.item.created_at}
+        serviceOrganizationName={this.getOrganizationName(
```

Comment:
Read above

Reply:
Fixed, i have split serviceOrganizationName in : serviceOrganizationName and serviceDepartmentName.